lib/utils/constants.js

```diff
@@ -81,11 +81,27 @@ module.exports.HOSTED_SKILL = {
         FAILURE: 'FAILED',
         IN_PROGRESS: 'IN_PROGRESS'
     },
-    GIT_HOOKS_TEMPLATES: {
+    S3_SCRIPTS: {
```

Comment:
Since we added DynamicConfig class. We should mode it there before merging.
https://github.com/alexa/ask-cli/pull/297

Something like this

`
    static get s3Scripts() {
        const baseUrl = resolve([process.env.ASK_S3_SCRIPTS_BASE_URL, 'https://s3-script-upload.s3-us-west-2.amazonaws.com']);
        return {
       authInfo: `${baseUrl}/auth_info`,
       prePush: `${baseUrl}/git-hooks-templates/pro/pre-push`,
       askPrePush: `${baseUrl}/git-hooks-templates/pro/ask-pre-push`,
       gitCredentialHelper: `${baseUrl}/helper/git-credential-helper`,
}


    }
`